Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:7
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install epel-release && \
+    yum -y install python2-pip && \
+    yum clean all
+
+ENV LANG=en_US.UTF-8
+ENV LANGUAGE=en_US.UTF-8
+ENV LC_ALL=en_US.UTF-8
+ENV PYTHONUNBUFFERED=0
+
+RUN pip install obal
```

Comment:
There's an issue here that this depends on the previous action: deployment to pypi. Chances are you're always building an old version in the container.